src/util/inconsistencies.py

```
@@ -80,7 +80,7 @@ def break_recursive_loops(cr, model, field, name_field="name"):
         name=SQL(get_value_or_en_translation(cr, table, name_field)),
     )
     cr.execute(update_query, [tuple(ids)])
-    bad_data = cr.fetchall()
+    bad_data = cr.fetchmany(100)
```

I'd instead raise an error if too many loops were found. When checking `cr.rowcount` above.